.github/workflows/ci.yml

```diff
@@ -844,11 +846,14 @@ jobs:
             build_tools/cmake/build_e2e_test_artifacts.sh \
             "${BUILD_E2E_TEST_ARTIFACTS_DIR}"
           echo "e2e-test-artifacts-dir=${BUILD_E2E_TEST_ARTIFACTS_DIR}/e2e_test_artifacts" >> "${GITHUB_OUTPUT}"
+          echo "e2e-test-artifacts-build-log=${BUILD_E2E_TEST_ARTIFACTS_DIR}/.ninja_log" >> "${GITHUB_OUTPUT}"
```

Comment:
So the MLIR compiler is multi-threaded. Won't our compilation time therefore be strongly affected by what else is going on on the machine if we just pull from the normal build log?

Reply:
It will, so our buildkite compilation time metric is noisy now. AFAIK they usually vary between +-20%. But we still caught multiple regressions when the compilation time surged over 100%.

It will be the next task to find the balance between parallelism and noise.